frontend/src/pages/modelRegistry/screens/ModelRegistry.tsx

```diff
@@ -30,7 +31,7 @@ const ModelRegistry: React.FC = () => {
         />
       }
       title={
-        <TitleWithIcon title="Deployed models" objectType={ProjectObjectType.deployedModels} />
+        <TitleWithIcon title="Registered models" objectType={ProjectObjectType.deployedModels} />
```

Comment:
@lucferbux this was previously changed to "Deployed models", can you check this?

Reply:
It was a mistake, @manaswinidas did the right thing here